packages/components/app/styles/components/dropdown.scss

```diff
@@ -413,8 +421,88 @@ $hds-dropdown-toggle-border-radius: 5px;
   }
 }
 
+// HDS::DROPDOWN::LIST-ITEM::TITLE
+
 .hds-dropdown-list-item--title {
   padding: 10px 16px 4px;
   color: var(--token-color-foreground-strong);
 }
 
+// HDS::DROPDOWN::LIST-ITEM::CHECKMARK
+
+.hds-dropdown-list-item--checkmark {
+  a.hds-dropdown-list-item__interactive,
+  button.hds-dropdown-list-item__interactive {
+    // disable background on focus and active states
+    &:focus,
+    &:active,
+    &.mock-focus,
+    &.mock-active {
+      color: var(--token-color-foreground-action-active);
+
+      &::after {
+        background-color: inherit;
+      }
+    }
+  }
+
+  .hds-dropdown-list-item__interactive-text {
+    flex: 1;
```

Comment:
have you considered moving this declaration directly to the `.hds-dropdown-list-item__interactive-text` instead of here? I _don't think_ there would be any unintended side effects, and would avoid having too many specific/custom declarations around (even the overrides of the `focus/active` states above make me nervous: was it _really_ necessary?).

Reply:
Indeed this property can be applied to the interactive text with no side effects.
Disabling the background color for focus/active states was needed to allow the use of badges inside the checkmark list item while [preserving a compliant contrast](https://www.figma.com/file/Kgc8BgB8jGuJXISndgTrD1/Dropdown?node-id=28361%3A64942&t=3jf3TMGNrOAbdpgZ-4).